test/cmd/admin.sh

```diff
@@ -481,4 +481,21 @@ os::cmd::expect_success_and_text 'oc describe identity test-idp:test-uid' 'test-
 os::cmd::expect_success_and_text 'oc describe user     test-cmd-user' 'test-idp:test-uid'
 os::test::junit::declare_suite_end
 
+# images
+os::test::junit::declare_suite_start "cmd/admin/images"
+
+# log in as an image-pruner and test that oadm prune images works against the atomic binary
+os::cmd::expect_success "oadm policy add-cluster-role-to-user system:image-pruner pruner --config='${MASTER_CONFIG_DIR}/admin.kubeconfig'"
+os::cmd::expect_success "oc login --server=${KUBERNETES_MASTER} --certificate-authority='${MASTER_CONFIG_DIR}/ca.crt' -u pruner -p anything"
+os::cmd::expect_success_and_text "oadm prune images" "Dry run enabled - no modifications will be made. Add --confirm to remove images"
+
+# import image and check its information
+os::cmd::expect_success "oc new-project pruner"
+os::cmd::expect_success "oc import-image busybox:latest --confirm -n pruner"
+os::cmd::expect_success_and_text "oadm top images" "pruner/busybox:latest"
+os::cmd::expect_success_and_text "oadm top imagestreams" "pruner/busybox"
```

Comment:
I don't want to have more aggressive checks here, since none of this information is stable enough. More agressive checks are done in unit testing.